twine-upload.sh

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
+set -Eeuo pipefail
+
```

Comment:
Should we do a sanity check like `if $INPUT_USER == __token__'`, then `$INPUT_PASSWORD` should start with `pypi-`?

Reply:
Yep.

Reply:
How about iterating on this later if it turns out that people use invalid tokens?

Reply:
I'm okay with pushing it to a separate PR. But I'd like to add this check anyway. We've seen people having such problems already.